distribution/src/main/resources/bin/elasticsearch.in.sh

```diff
@@ -86,3 +86,17 @@ JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 
 # Use our provided JNA always versus the system one
 JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
+
+# Try to figure out where to put error logs
+if [ "x$ERROR_DIR" = "x" ]; then
+  if [ "x$LOG_DIR" = "x" ]; then
+    ERROR_DIR="$ES_HOME/log"
```

Comment:
Hrm, should this be `$ES_HOME/logs`? Also, what about `path.logs`...it could be a sysprop, or also in `elasticsearch.yml`....